service/build.gradle

```diff
@@ -65,6 +65,10 @@ dependencies {
 
   // Documentation
   implementation "org.springdoc:springdoc-openapi-ui:${springDocOpenApiVersion}"
+
+  // mail
+  compile group: 'org.springframework.boot', name: 'spring-boot-starter-mail', version: '2.0.4.RELEASE'
```

Comment:
There is no need to pass the version. 
Please apply the correct pattern of dependency `compile "group:artifact"`